src/pybind/mgr/mgr_module.py

```diff
@@ -352,6 +352,10 @@ def CLIWriteCommand(prefix, args="", desc=""):
     return CLICommand(prefix, args, desc, "w")
 
 
+def _get_localized_key(prefix, key):
+    return '{}/{}'.format(prefix, key)
```

Comment:
Just as a totally optional suggestion (http://tracker.ceph.com/issues/38717): creating a `OptionsMixing` (mixin because it's using data/methods that are not defined) and moving therein this function and also the `get/set_module_options()`, so the common code is extracted and then consumed by inheritance (mixin-like).

```python
class OptionsMixing(object):
    ...
    def _get_localized_key(key):
        return '{}/{}'.format(self.get_mgr_id(), key)

    def get_config_option(..)
          ...

class MgrStandbyModule(ceph_module.BaseMgrStandbyModule, OptionsMixing):
    ...

class MgrModule(ceph_module.BaseMgrModule, OptionsMixing):
    ...
```